Sources/AppCoinsSDK/Domain/Repositories/Currency/CurrencyRepository.swift

```diff
@@ -17,11 +17,11 @@ internal class CurrencyRepository: CurrencyRepositoryProtocol {
         if let userCurrency = self.UserCurrency.getValue(forKey: "userCurrency") {
             completion(.success(userCurrency))
         } else {
-            AppCoinBillingService.convertCurrency(money: "1.0", fromCurrency: Coin.APPC, toCurrency: nil) { result in
+            AppCoinBillingService.convertCurrency(money: "1.0", fromCurrency: Currency.appcCurrency.currency, toCurrency: nil) { result in
                 switch result {
                 case .success(let userCurrencyRaw):
-                    self.UserCurrency.setValue(Currency(userCurrencyRaw: userCurrencyRaw), forKey: "userCurrency", storageOption: .memory)
-                    completion(.success(Currency(userCurrencyRaw: userCurrencyRaw)))
+                    self.UserCurrency.setValue(Currency(convertCurrencyRaw: userCurrencyRaw), forKey: "userCurrency", storageOption: .memory)
```

Comment:
Why are we using a convertCurrencyRaw initializer when what we have is a userCurrencyRaw?